lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -224,34 +238,26 @@ class Test__apply_params(IrisTest):
         warning_types=[UserWarning, DeprecationWarning, ImportWarning,
                        ImportWarning])
     def setUp(self):
-        """Use temperature cube to test with."""
-        data = (np.tile(np.linspace(-45.0, 45.0, 9), 3).reshape(3, 3, 3) +
-                273.15)
-        data[0] -= 2
-        data[1] += 2
-        data[2] += 4
-        data = data.astype(np.float32)
-        self.current_temperature_forecast_cube = set_up_variable_cube(
-            data, units="Kelvin", realizations=[0, 1, 2])
-
-        optimised_coeffs = [4.55819380e-06, -8.02401974e-09,
-                            1.66667055e+00, 1.00000011e+00]
-        current_cycle = "20171110T0000Z"
-        estimator = (
-            EstimateCoefficientsForEnsembleCalibration(
-                "gaussian", current_cycle, desired_units="Celsius"))
-        self.coeffs_from_mean = (
-            estimator.create_coefficients_cube(
-                optimised_coeffs, self.current_temperature_forecast_cube))
-
-        optimised_coeffs = np.array([5, 1, 0, 0.57, 0.6, 0.6])
-        estimator = (
-            EstimateCoefficientsForEnsembleCalibration(
-                "gaussian", current_cycle, desired_units="Celsius",
-                predictor_of_mean_flag="realizations"))
-        self.coeffs_from_realizations = (
-            estimator.create_coefficients_cube(
-                optimised_coeffs, self.current_temperature_forecast_cube))
+        """Set up expected arrays for the calibrated ensemble mean and variance
+        depending upon whether the ensemble mean or ensemble realizations have
+        been used."""
+        super().setUp()
+        self.expected_calibrated_predictor_mean = (
+            np.array([[273.7371, 274.6500, 275.4107],
+                      [276.8409, 277.6321, 278.3928],
+                      [279.4884, 280.1578, 280.9794]]))
+        self.expected_calibrated_variance_mean = (
+            np.array([[0.2134, 0.2158, 0.0127],
+                      [0.0247, 0.0215, 0.0127],
+                      [0.0581, 0.0032, 0.0008]]))
+        self.expected_calibrated_predictor_realizations = (
```

Comment:
It might be worth doing the statsmodel and no statsmodel versions of these tests, just for future reference.